imsv-docs-astro/src/content/docs/guides/regions.mdoc

```diff
@@ -1,55 +1,85 @@
 ---
 title: Supported Regions
 ---
-
 A crypto holder must reside in one of the supported regions to be able to
 order and use Immersve cards.
 
-The availability of certain regions may vary based on the partner card program:
-a region may be supported by Immersve but might not be accessible to a given
-partner if they have no card programs set up in that region.
-
-For the most up-to-date list of supported regions for each environment and for
-their availability to a specific partner account, please use the {% link
-endpoint="get-supported-regions" /%} endpoint.
-
 Below is an indicative list of regions supported by Immersve.
 
-## Live
+### Live
 - New Zealand
 - Australia
 
-## Coming soon
-- Austria
-- Belgium
-- Bulgaria
-- Croatia
-- Cyprus
-- Czech Republic (Czechia)
-- Denmark
-- Estonia
-- Finland
-- France
-- Germany
-- Greece
-- Hungary
-- Iceland
-- Ireland
-- Italy
-- Latvia
-- Liechtenstein
-- Lithuania
-- Luxembourg
-- Malta
-- Netherlands
-- Norway
-- Poland
-- Portugal
-- Romania
-- Slovakia
-- Slovenia
-- Spain
-- Sweden
-- United Arab Emirates
-- United Kingdom
-- United States
+### Coming soon
+Austria, Belgium, Bulgaria, Croatia, Cyprus, Czech Republic (Czechia), Denmark, Estonia, Finland, France, Germany, Greece, Hungary, Iceland, Ireland, Italy, Latvia, Liechtenstein, Lithuania, Luxembourg, Malta, Netherlands, Norway, Poland, Portugal, Romania, Slovakia, Slovenia, Spain, Sweden, United Arab Emirates, United Kingdom, United States
+
+## Supported Regions Endpoint
+
+The availability of regions may vary between partners. A region may be
+supported by Immersve but not be accessible to a given partner if they, for
+instance, have no card programs set up in that region.
+
+For the most up-to-date list of the supported regions for a partner, use the
+supported regions endpoint. This will provide a complete list of regions
+including:
+
+ 1. the region name
```

Comment:
not sure if it should be a numbered list, the order is not really important here. But I'm nitpicking :)